notebooks/Hugging_Face_VisionTranformer.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>Is it required to turn <code>y</code> into an ndarray here? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/skorch-dev/skorch/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Ah I see why it is required.